package.json

```diff
@@ -33,7 +33,7 @@
     "d3": "^4.7.4",
     "foam2": "git://github.com/foam-framework/foam2.git#confluence",
     "jquery": "^3.2.1",
```

Comment:
But we still directly depend on jquery here?

Reply:
Yes, we depended on two of them! Grepping for "jquery" shows only config/webpack.common.js using it, but if it's doing something there I wonder if it could be a dev dependency. @mdittmer can you advise?

Reply:
All it does is to provide jquery to the front-end, but I don't see anything actually using it. I suspect we can simply get rid of it.

Reply:
I'll do that in a separate PR so we can bisect any regression more easily.